### PR TITLE
Refactors the listing.html file and adds a heading to the page

### LINKED ISSUE #165: [Feature] Add a section heading to the resource cards
## Problem
I noticed there is no heading added to the resource cards on the homepage. This might give a user some kind of different random meaning or interpretation of which section it is or what the cards stand for. 

resourceCard

## Description
 I would like to add a section heading to the cards as it is seen on the featured projects in the [opencource.creativecommons.org](https://opensource.creativecommons.org/)

featuredResources

## Implementation
- [x ] I would be interested in implementing this feature.

### LINKED ISSUE #301: [Feature] Add proper documentation to listing.html and refactor for better structure
## Problem

The code in the `listing.html` file is not well-documented for a new user to understand and also, it needs better structure for understandability. formatting the file with `prettier` will help here.

## Description
Providing documentation and better structure will smoothen the process of understanding code for new users and maintainers.

## Implementation

- [x] I would be interested in implementing this feature.

### PR DESCRIPTION
**Pull request by Murdock9803**
## Fixes
- Fixes #301 by @Murdock9803 
- Fixes #165 by @KrownWealth
- Related to #101 by @Xaid-vfx 

## Description
The pull request aims at enhancing the `listing.html` file by providing proper documentation, formatting the code and also a heading for the page.

## Technical details
The PR performs the following tasks:
- Adds a heading to the page.
- Adds clear documentation about various sections in the file.
- Formats the code with `prettier` code formatter.

## Screenshots
Heading:
Screenshot 2024-07-04 031132

## Checklist

- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added or updated tests for the changes I made (if applicable).
- [x] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

For the purposes of this DCO, "license" is equivalent to "license or public domain dedication," and "open source license" is equivalent to "open content license or public domain dedication."
<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Murdock9803** [comment]
@possumbilities please have a look at the PR

File: docs/_layouts/listing.html [hunk 1]
```diff
@@ -4,61 +4,99 @@
...
 
   <body>
+    <!-- The standard CC header, located at docs/_includes/header.html -->
     {% include header.html %}
 
     <main>
-      {{content}}
+      <!-- The main heading for the site -->
+      <header>
+        <h1>RESOURCES</h1>
```
Comment (possumbilities) [inline docs/_layouts/listing.html:41]: this should be standard case `Resources`, and made uppercase with CSS.

Comment (Murdock9803) [reply docs/_layouts/listing.html]: @possumbilities I have corrected the heading text format, and it is now according to the standard case 🙌

**Review by possumbilities — APPROVED** [review]
Approved!

**Pull request merged**